device_services/generic/meter.md

```diff
@@ -17,17 +17,21 @@ An electricity meter service can represent a stand-alone AMS meter, like a HAN s
+| in   | cmd.meter.reset                  | null       |                                          |             |             | Resets all historical readings.                                                                                                                                                     |
+| -    |                                  |            |                                          |             |             |                                                                                                                                                                                     |
+| in   | cmd.meter_export.get_report      | string     |                                          |             |             | Value is a export/production `unit` defined in [`sup_export_units`](#service-properties) property.                                                                                  |
+| out  | evt.meter_export.report          | float      | `unit`, `prv_data`, `delta_t`            | `aggregate` | `unit`      | Returns a export/production meter report for `unit` specified in properties.                                                                                                        |
+| -    |                                  |            |                                          |             |             |                                                                                                                                                                                     |
+| in   | cmd.meter_ext.get_report         | str_array  |                                          |             |             | Requests an extended electricity report for listed extended values. Empty or null value requests all supported extended values.                                                |
+| out  | evt.meter_ext.report             | float_map  |                                          | `split`     |             | Returns an extended electricity report. See [electricity measurements](#electricity-measurements) section for more information.                                                    |
+| -    |                                  |            |                                          |             |             |
+| in   | cmd.meter_transformer.get_report | null       |                                          |             |             | Requests a list of transformation ratios for each measurement affected by the transformer.                                                                                                               |
+| out  | evt.meter_transformer.report     | float_map  |                                          | `split`     |             | Returns a map of transformation ratios for each measurement affected by the transformer.                                                                                            |
```

Comment:
no point to have storage strategy split here, we can assume that report will always contain ALL units which are transformed.